source/mongoshrc.txt

```diff
@@ -18,35 +18,9 @@
 
 .. include:: /includes/fact-mongoshrc-loading.rst
 
-Examples
---------
-
-The following examples create custom prompts. To display:
-
-- :ref:`line numbers <ex-display-line-numbers>`
-- :ref:`database and hostname <ex-display-dbname-and-hostname>`
-- :ref:`system uptime and document count <ex-display-system-up-time>`
-
-add the example code to :ref:`.mongoshrc.js <mongoshrc-js>`.
-
-.. _ex-display-line-numbers:
-
-Display Line Numbers
-~~~~~~~~~~~~~~~~~~~~
-
-.. include:: /includes/examples/ex-display-line-numbers.rst
-
-.. _ex-display-dbname-and-hostname:
-
-Display Database and Hostname
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. include:: /includes/examples/ex-display-dbname-and-hostname.rst
-
-.. _ex-display-system-up-time:
-
-Display System Up Time and Document Count
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. include:: /includes/examples/ex-display-system-uptime.rst
+Example Usage
+-------------
 
+You can use the .mongoshrc file to customize ``mongosh`` startup
+behavior and add scripted functionality. To see examples, see
+:ref:`mongosh-write-scripts-config-file`.
```

Comment:
Thanks - I found the ref anchor.

GH didn't show the file diff by default so ctrl+F couldn't find the other mongosh-write-scripts-config-file on this page.